src/Engine/ProtoCore/Namespace/SymbolTable.cs

```diff
@@ -438,7 +438,8 @@ public int GetSymbolCount()
         /// <returns>Array of matching symbols or empty array</returns>
         public Symbol[] TryGetSymbols(string name, Func<Symbol, bool> predicate)
         {
-            string symbolName = name.Split('.').Last();
+            var index = name.LastIndexOf('.');
```

Comment:
whats with this change - I might have missed it in the PR description?

Reply:
Ah sorry... just faster implementation.  Split finds all the dots and LastIndexOf moves from the back